spec/uk_new_secondary_completion_spec.rb

```diff
@@ -0,0 +1,25 @@
+require "rails_helper"
+
+RSpec.describe UkNewSecondaryCompletion do
+  let(:confirmed) { build(:uk_new_secondary_completion, :confirmed) }
+  let(:unconfirmed) { build(:uk_new_secondary_completion, :unconfirmed) }
+  let(:invalid) { build(:uk_new_secondary_completion, confirmed: nil) }
+
+  describe "validation" do
+    it "requires confirmed/unconfirmed" do
+      expect(confirmed).to be_valid
+      expect(unconfirmed).to be_valid
+      expect(invalid).to_not be_valid
+    end
+  end
+
+  describe "#next_step" do
+    it "returns opt_in_emails when confirmed" do
+      expect(confirmed.next_step).to eq("opt_in_emails")
```

Comment:
where is this step?

Reply:
I think these might be old steps we no longer use, I will have a look and maybe raise a PR to remove them